node-packages/commons/src/tasks.ts

```diff
@@ -776,7 +760,7 @@ export const createDeployTask = async function(deployData: any) {
     pullrequestTitle
   } = deployData;
 
-  const project = await getActiveSystemForProject(projectName, 'Deploy');
+  const project = await getOpenShiftInfoForProject(projectName);
```

Comment:
`getOpenShiftInfoForProject` return payload returns `{project: {project: { id, name, etc... }}}`

Which means you need to do [something like this](https://github.com/uselagoon/lagoon/blob/deprecate-activesystems/node-packages/commons/src/tasks.ts#L364-L365) to be able to then access the actual project data properly.

Maybe its worth fixing `getOpenShiftInfoForProject` so that it only returns the project data, rather than needing to extract the bit thats actually needed afterwards. I don't care either way, but if you change the response, you'll need to update other places that consume it then

Reply:
It is used in quite a few places though